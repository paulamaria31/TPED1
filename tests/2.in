10
XOVOXVVOV
VOXOXXOOX
XXXVVVOVO
OVOVXVXVV
XVVOVVOXV
XXXXXVXOV
VXXOOVVOV
OXOOVOXXV
XXOXOVXOO
XXOVVXXOO
