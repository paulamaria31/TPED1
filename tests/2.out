Tabuleiro 1 em andamento [X: (3,3)]
Tabuleiro 2 com vitoria [X]
Tabuleiro 3 com vitoria [X]
Tabuleiro 4 em andamento [proximo jogador indefinido]
Tabuleiro 5 em andamento [proximo jogador indefinido]
Tabuleiro 6 invalido
Tabuleiro 7 em andamento [X: (1,1)]
Tabuleiro 8 em andamento [X: (3,3)(2,2)]
Tabuleiro 9 com vitoria [X]
Tabuleiro 10 em andamento [O: sem jogada mestre]
